src/NuGet/Microsoft.Net.Compilers.Toolset/CoreClrCompilerArtifacts.targets

```diff
@@ -10,6 +10,8 @@
 
       <CoreClrCompilerBinArtifact Include="$(ArtifactsBinDir)Microsoft.CodeAnalysis\$(Configuration)\net6.0\Microsoft.CodeAnalysis.dll" />
       <CoreClrCompilerBinArtifact Include="$(ArtifactsBinDir)Microsoft.CodeAnalysis\$(Configuration)\net6.0\**\Microsoft.CodeAnalysis.resources.dll" />
+      
+      <CoreClrCompilerBinArtifact Include="$(ArtifactsBinDir)Microsoft.CodeAnalysis.Private\$(Configuration)\net6.0\Microsoft.CodeAnalysis.Private.dll" />
```

Comment:
@dotnet/roslyn-compiler this should be run through RPS before merging. I think it's going to trigger an assembly load regression, and I'm not sure the benefits of this change will warrant approving the new assembly. If it does get pre-approval, obviously no reason to block on this aspect.

Reply:
@jjonescz I guess it will be less noisy to go back to the NuGet approach. I wasn't aware that adding a new dll is so tricky.

@sharwell does that work well?

Reply:
I'm not sure what the NuGet approach is

Reply:
That wouldn't make any difference. The test looks at assemblies loaded in VS, regardless of source location.